src/sql/src/plan/statement/show.rs

```diff
@@ -595,13 +595,24 @@ pub fn show_columns<'a>(
     )
 }
 
+// The rationale for which fields to include in the tuples are those
+// that are mandatory when creating a replica as part of the CREATE
+// CLUSTER command, i.e., name and size.
 pub fn show_clusters<'a>(
     scx: &'a StatementContext<'a>,
     filter: Option<ShowStatementFilter<Aug>>,
 ) -> Result<ShowSelect<'a>, PlanError> {
-    let query = "SELECT mz_clusters.name FROM mz_catalog.mz_clusters".to_string();
-
-    ShowSelect::new(scx, query, filter, None, Some(&["name"]))
+    let query = "
+SELECT
+    mc.name,
+    pg_catalog.string_agg(mcr.name || ' (' || mcr.size || ')', ', ' ORDER BY mcr.name)
```

Comment:
I still think this would be better as a json array than a string. Cluster names with commas or parens could make this hard to parse, whereas if this were a json array of objects with fields name and size, it becomes trivial.

Reply:
Hard to parse for humans or for computers? @parkerhendo and I were coalescing above around a philosophy that `SHOW` commands are meant for humans staring at their terminals, not computers. (Computers already have a great interface, in the highly normalized schema in `mz_catalog`.)

For me, using JSON arrays doesn't pass the readability test for humans!

```
SHOW REPLICAS;
 cluster |                               replicas                               
---------+----------------------------------------------------------------------
 c1      | [{"name": "r1", "size": "small"}, {"name": "r2", "size": "2xsmall"}]
 c2      | [{"name": "r2", "size": "medium"}]
```

Reply:
SGTM.